CHANGELOG.md

```diff
@@ -1,11 +1,25 @@
 # CHANGELOG
 
+## 3.1.0
+#### New Functionality
+
+- Support for Optic-update Apis.
+- New endpoint for GraphQL.
+- Support for calling any REST endpoint with Node Client.
+- Support to refresh expired token when using MarkLogic cloud authentication.
```

Comment:
I'm being picky, but I like to avoid "Supports for" or "Support to" in favor of a more active description - i.e. "Tokens are automatically refreshed before expiration when using MarkLogic Cloud authentication."

That makes it clear too that the client is doing the work, whereas "support for" implies that a user may need to do something to leverage the new "support".